src/aws_encryption_sdk/keyrings/raw.py

```diff
@@ -240,19 +219,18 @@ class RawRSAKeyring(Keyring):
     """Generate an instance of Raw RSA Keyring which performs asymmetric encryption and decryption using public
     and private keys provided
 
+    .. versionadded:: 1.5.0
+
     :param str key_namespace: String defining the keyring ID
     :param bytes key_name: Key ID
-    :param private_wrapping_key: Private encryption key with which to wrap plaintext data key (optional)
-    :type private_wrapping_key: RSAPrivateKey
-    :param public_wrapping_key: Public encryption key with which to wrap plaintext data key (optional)
-    :type public_wrapping_key: RSAPublicKey
-    :param wrapping_algorithm: Wrapping Algorithm with which to wrap plaintext data key
-    :type wrapping_algorithm: WrappingAlgorithm
-    :param key_provider: Complete information about the key in the keyring
-    :type key_provider: MasterKeyInfo
+    :param RSAPrivateKey private_wrapping_key: Private encryption key with which to wrap plaintext data key (optional)
+    :param RSAPublicKey public_wrapping_key: Public encryption key with which to wrap plaintext data key (optional)
+    :param WrappingAlgorithm wrapping_algorithm: Wrapping Algorithm with which to wrap plaintext data key
+    :param MasterKeyInfo key_provider: Complete information about the key in the keyring
 
     .. note::
-    At least one of public wrapping key or private wrapping key must be provided.
+
+       At least one of public wrapping key or private wrapping key must be provided.
```

Comment:
nit: three spaces should be four

Reply:
Unfortunately not; Python is 4-space indent; rst is 3-space indent.

Reply:
I stand corrected; I checked this while tweaking some other aspects of the docs and 4-space works here as well. There are some places in rst that require 3-space, but it looks like this is not one of them.